File: Examples/VisualBasic/Articles/DeleteBlankRowsColumns/DeletingBlankRows.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Cells

Namespace Articles.DeleteBlankRowsColumns
    Public Class DeletingBlankRows
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Open an existing excel file.
            Dim wb As New Workbook(dataDir & "SampleInput.xlsx")

            ' Create a Worksheets object with reference to
            ' The sheets of the Workbook.
            Dim sheets As WorksheetCollection = wb.Worksheets

            ' Get first Worksheet from WorksheetCollection
            Dim sheet As Worksheet = sheets(0)

            ' Delete the Blank Rows from the worksheet
            sheet.Cells.DeleteBlankRows()

            ' Save the excel file.
            wb.Save(dataDir & "output.xlsx")
            ' ExEnd:1


        End Sub
    End Class
End Namespace

File: Examples/CSharp/Articles/DeleteBlankRowsColumns/DeletingBlankRows.cs
using System;
using System.IO;

using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles.DeleteBlankRowsColumns
{
    public class DeletingBlankRows
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Open an existing excel file.
            Workbook wb = new Workbook(sourceDir + "sampleDeletingBlankRows.xlsx");

            // Create a Worksheets object with reference to
            // The sheets of the Workbook.
            WorksheetCollection sheets = wb.Worksheets;

            // Get first Worksheet from WorksheetCollection
            Worksheet sheet = sheets[0];

            // Delete the Blank Rows from the worksheet
            sheet.Cells.DeleteBlankRows();

            // Save the excel file.
            wb.Save(outputDir + "outputDeletingBlankRows.xlsx");

            Console.WriteLine("DeletingBlankRows executed successfully.");
        }
    }
}
